src/Microdown-RichTextPresenter/MicMorphicTextAdapter.class.st

```diff
@@ -56,8 +90,14 @@ MicMorphicTextAdapter >> selectionInterval [
 { #category : #'presenter->morph' }
 MicMorphicTextAdapter >> selectionInterval: anInterval [
 	"called by presenter to set selection in morph"
-	self widgetDo: [ :w | w setTextAreaSelection: anInterval ].
-	
+
+	self widgetDo: [ :w | w setTextAreaSelection: anInterval ]
+]
+
+{ #category : #factory }
+MicMorphicTextAdapter >> setEnabled: aBoolean to: aWidget [
```

Comment:
I do not see the value of this method.
`aWidget enabled: aBoolean` is more clear

Reply:
I just adapted to what I have in SpMorphicTextAdapter. Is maybe useless, but it does not hurts.